test/models/email_configuration_test.rb

```diff
@@ -45,7 +45,7 @@ class EmailConfigurationTest < ActiveSupport::TestCase
       assert_equal local_settings[attr], config.smtp_settings[attr]
     end
 
-    %i[enable_starttls_auto enable_starttls tls port address openssl_verify_mode].each do |attr|
+    %i[enable_starttls_auto enable_starttls port address openssl_verify_mode].each do |attr|
```

Comment:
why remove this option?

Reply:
We don't have `tls` option in the SMTP config https://github.com/3scale/porta/blob/master/openshift/system/config/smtp.yml

Reply:
But OK, we can bring it back :grimacing:

Reply:
Not all options are strictly needed and some take precedence.